static/app/views/projectDetail/projectDetail.tsx

```diff
@@ -347,13 +348,17 @@ const StyledPageContent = styled(PageContent)`
   padding: 0;
 `;
 
+const Body = styled(Layout.Body)`
+  padding-top: ${space(2)} !important;
```

Comment:
Oof, kinda wish we didn't have to do this

Reply:
updated on the thirds component instead to adjust the padding